peachjam/migrations/0132_outcome_es_index.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.21 on 2024-05-15 13:59
+
+import os
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    from django.conf import settings
+    from django_elasticsearch_dsl.registries import registry
+
+    # if not settings.DEBUG and os.environ.get("ELASTICSEARCH_HOST"):
+    for ix in registry.get_indices():
+        if not ix._mapping:
+            continue
+        print(f"Adding outcome mapping for {ix._name}")
+        ix.connection.indices.put_mapping(
+            index=ix._name,
+            body={
+                "properties": {
```

Comment:
@longhotsummer I also noticed that your code did not include the "properties" keyword. I get errors when I omit it